src/ln/channelmanager.rs

```diff
@@ -22,7 +22,7 @@ use secp256k1;
 
 use chain::chaininterface::{BroadcasterInterface,ChainListener,ChainWatchInterface,FeeEstimator};
 use chain::transaction::OutPoint;
-use ln::channel::{Channel, ChannelError};
+use ln::channel::{Channel, ChannelError, COMMITMENT_TX_BASE_WEIGHT, COMMITMENT_TX_WEIGHT_PER_HTLC};
```

Comment:
This results in unused import warnings - you should just add these imports inside the tests module and drop the "super::" paths where they're used.

Reply:
Done.